packages/arb-token-bridge-ui/src/util/fastBridges.ts

```diff
@@ -7,7 +7,6 @@ import Across from '@/images/bridge/across.png'
 import Stargate from '@/images/bridge/stargate.png'
 import Synapse from '@/images/bridge/synapse.png'
 import Wormhole from '@/images/bridge/wormhole.svg'
-import LIFI from '@/images/bridge/lifi.webp'
```

Comment:
Maybe we should comment this one out for now instead. It's part of the commented out code after we've temporarily hidden LIFI from the fast bridge list